module/attribute/src/Infrastructure/Grid/AttributeGroupGrid.php

```diff
@@ -28,14 +28,12 @@ class AttributeGroupGrid extends AbstractGrid
      */
     public function init(GridConfigurationInterface $configuration, Language $language): void
     {
-        $filters = $configuration->getFilters();
-
-        $id = new TextColumn('id', 'Id', new TextFilter($filters->get('id')));
+        $id = new TextColumn('id', 'Id', new TextFilter());
         $id->setVisible(false);
         $this->addColumn('id', $id);
-        $this->addColumn('code', new TextColumn('code', 'Code', new TextFilter($filters->get('code'))));
-        $this->addColumn('name', new TextColumn('name', 'Name', new TextFilter($filters->get('name'))));
-        $this->addColumn('elements_count', new IntegerColumn('elements_count', 'Elements Count', new TextFilter($filters->get('elements_count'))));
+        $this->addColumn('code', new TextColumn('code', 'Code', new TextFilter()));
```

Comment:
In TextColum u can use TextFilter as class default. Less coding :)

Reply:
column don't required any filter then shouldn't have any default one